streamlit-chatbot-app/requirements.txt

```diff
@@ -1 +1,2 @@
-openai
\ No newline at end of file
+mlflow
+streamlit
```

Comment:
Can we try to pin the library versions in the `requirements.txt` files across the templates? That would help the templates be stable across time. I've actually seen some templates break when restarted because the libraries were not pinned.

Reply:
You could inspect the app logs to see what library versions were used.